### ISSUE TITLE
Investigate workaround for screen readers not always announcing the selected file after picking

### ISSUE THREAD

**Issue by romaricpascal** [alphagov/govuk-frontend#5684]
## What

Investigate workaround for screen readers not always announcing the selected file after picking.

A starting point could be this [Slack message (internal)](https://gds.slack.com/archives/C07D4RK4GJU/p1738263031615179) mentionning how screen readers buffer their announcements and we may want to delay focusing the button after coming back to the page by a couple of seconds.

## Why

Currently screen readers sometimes lose the announcement of the selected file name (or 'N files selected') when focus comes back to the page with a file picker.

## Who needs to work on this

Developers

## Who needs to review this

Developers, Accessibility specialist if poss.

## Done when

- [ ] We spent 1/2-1 day investigating the issue and decided on a next step

**Comment by romaricpascal** [comment]
Issue is visible using JAWS and Chrome and happens:
- When clicking the field to select a file, leaving the announcement to be the initial state of the component (note JAWS announcing 'No File Chosen' instead of the visible 'png-…-thumbnail.png' filename):

<img width="1499" alt="Image" src="https://github.com/user-attachments/assets/58c9c28e-d6de-4968-95d3-2adf69b873f2" />

- When keeping the focus on the element and choosing a different file after having already selected one (note JAWS announcing 'Reader_en_install.exe' instead of the visible 'png-…-thumbnail.png' filename):

<img width="1497" alt="Image" src="https://github.com/user-attachments/assets/c9df2b21-eb83-4585-8a57-6d94e12070ab" />

**Comment by romaricpascal** [comment]
Fixed by #5729
